packages/flutter/test/widgets/fade_in_image_test.dart

```diff
@@ -84,6 +84,10 @@ FadeInImageParts findFadeInImage(WidgetTester tester) {
   }
 }
 
+class ClosureTracker {
```

Comment:
You don't really need an extra class for this. Just define a bool variable outside the closure, initialize it with false and set it to true within the closure.

Reply:
Hmm I tried this initially and it didnt seem to work, I probably did it wrong. I'll try again.

Reply:
Ok it works now, idk what I did earlier for this to not work..